setup.py

```diff
@@ -68,8 +68,9 @@
     license='MIT',
     scripts=[],
     data_files=[],
-    setup_requires=["numpy>=1.20", "cython>=0.23"],
+    setup_requires=["oldest-supported-numpy", "cython>=0.23"],
     install_requires=["numpy>=1.20", "scipy>=1.0"],
```

Comment:
```suggestion
    install_requires=["numpy>=1.16", "scipy>=1.0"],
```

I agree we should go back to the old numpy requirement